src/core/components/definitionList/definitionListItem/definitionListItem.tsx

```diff
@@ -14,15 +14,15 @@ export const DefinitionListItem: React.FC<IDefinitionListItemProps> = (props) =>
     return (
         <div
             className={classNames(
-                'flex w-full flex-col items-baseline justify-between gap-y-2 border-b border-neutral-100 py-3 last:border-none md:flex md:flex-row md:gap-x-6 md:py-4',
+                'flex flex-col gap-y-2 border-b border-neutral-100 py-3 last:border-none md:grid md:grid-cols-[1fr_2fr] md:gap-x-6 md:py-4',
                 className,
             )}
             {...otherProps}
         >
-            <dt className="line-clamp-1 shrink-0 text-base font-normal leading-relaxed text-neutral-800 md:line-clamp-6 md:w-40">
+            <dt className="line-clamp-1 text-base font-normal leading-relaxed text-neutral-800 md:line-clamp-none">
                 {term}
             </dt>
-            <dd className="size-full min-w-0 text-base font-normal leading-relaxed text-neutral-800">{children}</dd>
+            <dd className="text-base font-normal text-neutral-500">{children}</dd>
```

Comment:
As you're making changes here, could you add a story under the `DefinitionList.Item` component to showcase the usage of the item being a `<Link />` component? We had issues of links not being truncated. If we need to make changes to the link component in order to make it properly truncate, you can ignore this comment and we'll fix it on another ticket 👌🏻